src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -1238,6 +1238,8 @@ public virtual MethodSymbol TupleUnderlyingMethod
             }
         }
 
+        public bool HasBody => !IsAbstract && !IsExternal && (ContainingType == null || !ContainingType.IsComImport);
```

Comment:
>HasBody [](start = 20, length = 7)

Please add explicit implementation of Cci.IMethodDefinition.HasBody, which delegates to this property, in MethodSymbolAdapter.cs (with appropriate asserts, see implementation for other IMethodDefinition members)